Assets/Scripts/ObjectSpawning/EnemySpawner.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemySpawner : MonoBehaviour
+{
+    Spawner spawner;
+    public List<RoomNode> rooms;
+    public int difficulty;
+    public List<int> indexes;
+    static int enemyCount;
```

Comment:
Enemy count should not be static since it is incremented later in the code.

Reply:
done x2! thank you